forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt

```diff
@@ -106,6 +119,10 @@ class ForagePINEditText @JvmOverloads constructor(
         addView(getLogoImageViewLayout(context))
     }
 
+    internal fun getVaultType(): String {
+        return vaultType
+    }
```

Comment:
is this going to be used by the consumer of the SDK? I think it's safe to remove this.